ananas/src/main/java/iamutkarshtiwari/github/io/ananas/editimage/fragment/TextEditorDialogFragment.java

```diff
@@ -91,10 +91,10 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         LinearLayoutManager layoutManager = new LinearLayoutManager(getActivity(), LinearLayoutManager.HORIZONTAL, false);
         addTextColorPickerRecyclerView.setLayoutManager(layoutManager);
         addTextColorPickerRecyclerView.setHasFixedSize(true);
-        ColorPickerAdapter colorPickerAdapter = new ColorPickerAdapter(getContext());
+        ColorPickerAdapter colorPickerAdapter = new ColorPickerAdapter(getContext() , 0);
 
         //This listener will change the text color when clicked on any color from picker
-        colorPickerAdapter.setOnColorPickerClickListener(colorCode -> {
+        colorPickerAdapter.setOnColorPickerClickListener((colorCode, pos) -> {
```

Comment:
if this is the only implementation of this interface then we don't really need the extra parameter, do we?

Reply:
This `listener` is also used in `BrushConfigDialog.kt`. In there I'm utilising this position